### DISCUSSION TITLE
Preciso de um tester

### DISCUSSION THREAD

**Discussion by rodrigogregorioneri** [rodrigogregorioneri/serverless_integration#4] [Q&A] [answered]
Preciso de um tester

**Comment by neriidev** [comment, marked as answer]
Posso ser seu tester

**Reply by rodrigogregorioneri** [reply]
Show qual sua experiencia

**Reply by neriidev** [reply]
tenho experiencia com testes de integração, testes automatizados com selenium

**Reply by rodrigogregorioneri** [reply]
Show interessante
podemos marcar uma call?+
